CONTRIBUTING.md

```diff
@@ -23,6 +23,20 @@ git clone git@github.com:mermaid-js/mermaid.git
 cd mermaid
 # npx is required for first install as volta support for pnpm is not added yet.
 npx pnpm install
+pnpm -w run lint:fix
```

Comment:
`lint:fix` is unnecessary when running for the first time as the CI already lints the branches.

Reply:
Nope, I faced the issue immediately after fork, I've made a new branch from current development branch, but `pnpm test` failed. That was the command to resolve it, so I think it must be in guidelines.

Reply:
Did you note which files changed after running the command? 
Also, what error was thrown during the test?

Reply:
Fixed